mysql/azuremysql/azurecertfetcher.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2018 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package azuremysql provides connections to Azure Database for MySql.
+package azuremysql // import "gocloud.dev/mysql/azuremysql"
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"golang.org/x/net/context/ctxhttp"
+)
+
+type (
+
+	// AzureCertFetcher loads the CA certificates from remote endpoint or file.
+	AzureCertFetcher struct {
+		client *http.Client
```

Comment:
`Client` should be exported so as to permit the user to make changes.

Reply:
Done